manifest.json

```diff
@@ -4067,52 +4067,6 @@
                 }
             }
         },
-        "net.LeCloutPanda.InspectorCustomizer": {
-            "name": "InspectorCustomizer",
-            "description": "Allows you to customize some Neos panel items including Inspector, Logix browser and more.",
-            "category": "General UI Tweaks",
-            "sourceLocation": "https://github.com/LeCloutPanda/Inspector-Customizer",
```

Comment:
Previously when a mod has been deprecated we've just been asking people to add the `deprecated` flag... but I see you've nuked the whole repo. I suppose in a case like this removing the entry is okay? Curious what the other manifest maintainers think.

Reply:
I think that it would be fine to remove the entry if the repo gets nuked as it becomes redundant to have a link that leads to no where and in my case I had moved "Inspector-Customizer" to my "Customise" line of mods

Reply:
Ah, I see. I hadn't made it that far down the diff, yet.